examples/InspectLibraries/main.go

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) nexB Inc. and others. All rights reserved.
+// ScanCode is a trademark of nexB Inc.
+// SPDX-License-Identifier: Apache-2.0
+// See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+// See https://github.com/nexB/go-inspector for support or download.
+// See https://aboutcode.org for more information about nexB OSS projects.
+//
+
+package main
+
+import (
+	"fmt"
+	goinspector "github.com/nexB/go-inspector"
```

Comment:
I guess we should rename the repo to github.com/nexB/go_inspector for simplicity? In anycase avoid using go_inspector in someplace and goinspector elsewhere.

Reply:
@pombredanne,  you can change the name of the repository